README.md

```diff
@@ -56,6 +56,7 @@ path|description
 **Train**
 path|compute|environment|description
 -|-|-|-
+[examples/train/deepspeed-cifar.py](examples/train/deepspeed-cifar.py)|unknown|docker|train CIFAR-10 using DeepSpeed and PyTorch
```

Comment:
rerunning `readme.py` after adjusting the compute target will fix the "unknown" compute here

Reply:
Fixed.